packages/commonwealth/client/scripts/views/pages/view_thread/ViewThreadPage.tsx

```diff
@@ -1,5 +1,5 @@
 import { ContentType, getThreadUrl, slugify } from '@hicommonwealth/shared';
-import axios from 'axios';
+import useGetViewCountByObjectIdQuery from 'client/scripts/state/api/general/getViewCountByObjectId';
```

Comment:
maybe you will be able to change the setting in your IDE so not adding `client/scripts` prefix to the imports?